InvoiceNumbering/Reactor.fs

```diff
@@ -0,0 +1,14 @@
+module InvoiceNumberingReactor
+
+type Service(numberService: InvoiceNumbering.Service, invoiceService: Invoice.Service) =
+  member _.Handle(streamName, event) =
+    async {
+      match streamName with
+      | FsCodec.StreamName.CategoryAndId(Invoice.Category, Invoice.InvoiceId.Parse invoiceId) ->
```

Comment:
I guess this is not the time to keep it clear, but you could add a `let (|StreamName|_|) ...` to put the tryparse beside the gen in the aggregate's module